catalog/transformer/catalog-transformer-rtf/pom.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>transformer</artifactId>
+        <groupId>ddf.catalog.transformer</groupId>
+        <version>2.14.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <name>DDF :: Catalog :: Transformer :: RTF</name>
+    <packaging>bundle</packaging>
+    <artifactId>catalog-transformer-rtf</artifactId>
+
+    <properties>
+        <jrtf.version>0.1.0</jrtf.version>
```

Comment:
❓ Should we define this in the project's root pom instead?

Reply:
Yes if anything else was using this library, but as it stands this is the only place so I would be inclined to just leave it here.

Reply:
Localization sounds good to me